### DISCUSSION TITLE
Adding An Apartment To CoreMasterBundle (within ExampleAssets.unitypackage)

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3757] [Ideas]
Apologizes for making so many discussions. Currently the file setup of the materials used in material palettes are within the package (Objects/Material_Palettes), but there is no apartment within the package that utilizes this. When I first bumped into this issue it caused me a headache due to it raising questions like: Does the palette material only apply to gameobjects named a certain way?, Does the original material of the object have to be named a certain way for it to work?, etc. Adding one of the apartments to the package would definitely solve almost all of these common questions.

**Comment by SDGNelson** [comment]
It doesn't need any special naming, the material is replaced on all child renderer components.

**Reply by Jdance-Media** [reply]
Thanks 👍
